cmd/tsbs_generate_data/serialize/siridb.go

```diff
@@ -0,0 +1,70 @@
+package serialize
+
+import (
+	"encoding/binary"
+	"fmt"
+	"io"
+	"log"
+	"strconv"
+
+	qpack "github.com/transceptor-technology/go-qpack"
+)
+
+// SiriDBSerializer writes a Point in a serialized form for TimescaleDB
+type SiriDBSerializer struct{}
```

Comment:
This file has several left over/outdated comments referring to TimescaleDB rather that the actual form of the data stored.